app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Auth;
+use App\Models\User;
+class UserController extends Controller
+{
+    //
+    public function login(Request $req)
+    {
+        if(empty($req->user) && empty($req->password)){
+            $error = "Email or Password must be not empty!";
+            return View('login',['error'=>$error]);
+        }
+        $user = User::where('email','=',$req->email)->first();
```

Comment:
không có toán tử '=' thì có chạy được không

Reply:
có anh ơi